pages/index.js

```diff
@@ -232,7 +232,7 @@ export default function Home({
                 <FaDollarSign /> Free Tier Forever
               </div>
             </div>
-            <div className="mt-10 flex flex-col sm:flex-row items-center justify-center gap-6 mx-24">
+            <div className="mt-10 flex flex-col sm:flex-row items-center justify-center gap-6 sm:mx-24">
```

Comment:
how about making it more dynamic, like this instead of a static value
```suggestion
            <div className="mt-10 flex flex-col sm:flex-row items-center justify-center gap-6 min-w-[300px] max-w-[700px] w-[50%]">
```

Reply:
Yes I agree, it is never good to add px

Reply:
Hi, the suggested changes leads to the below output:
![image](https://github.com/EddieHubCommunity/BioDrop/assets/111000515/11930633-6b8b-4b7f-a09e-cf12517e8af2)

The issue is that `min-width: 300px` will cause the website UI to overflow for lower screen width devices like samsung galaxy fold (Screen width: 280px) etc.

`max-width: 700px` does nothing as the container / wrapper of the button layout already has the max-width property.

`width: 50%` will cause useability issues on lower mobile devices.
The current code is fully responsive and I don't find any reasons to implement the suggest changes.
Please let me know if I have missed something.

Reply:
I have marked as this as `unresolved` as it is still an ongoing discussion.

Reply:
After reviewing the PR, I think `sm:` does work better here